package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "gulp-html",
-  "version": "0.4.7",
+  "version": "0.4.4",
```

Comment:
could you keep this version?

Reply:
Yes, sure.

Reply:
Please clarify, what do you mean?

Reply:
I meant this version should be bumped by `npm version` instead. So we need to keep the prev version `0.4.7` here.

Reply:
@watilde just changed to 0.4.7.